docs/reference/commandline/run.md

```diff
@@ -253,6 +254,15 @@ Docker daemon.
 
 For in-depth information about volumes, refer to [manage data in containers](https://docs.docker.com/engine/tutorials/dockervolumes/)
 
+### Mount using the new interface (--mount)
+
+The new mount interface is described in the [service create command section](service_create.md).
+
+Examples:
+
+    $ docker run --read-only --mount type=volume,target=/icanwrite busybox touch /icanwrite/here
+    $ docker run -t -i --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock busybox sh
```

Comment:
done. now `type=bind,src=/data,dst=/data`.

In my understanding this PR doesn't have an impact on API (doc has notation about [`Mounts](https://github.com/docker/docker/blob/047e44eeb186a8bde41765a44c5966a150fa84f3/docs/reference/api/docker_remote_api_v1.25.md#create-a-container), already), so I didn't touch `docs/reference/api` in this PR.